frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -38,45 +38,44 @@ function isValidClusterNameFormat(name: string) {
 }
 
 export default function SettingsCluster() {
-  const cluster = useCluster();
   const clusterConf = useClustersConf();
+  const clusters = Object.keys(clusterConf || {});
```

Comment:
Although the cluster keys may equal the cluster name, this should rather be the name of each cluster (i.e. name from the each cluster value).